config/initializers/menus.rb

```diff
@@ -42,7 +42,7 @@
             last: true
   menu.push :help, OpenProject::Info.help_url,
             last: true,
-            caption: I18n.t('label_help'),
+            caption: '',
```

Comment:
As we only need to set the html title, I argue that allowing an override in the html options is the simplest solution. I have updated the PR to show the simplicity of that solution.

The title is internally set to `html_options[:title]` [(cf. menu_helper.rb)](https://github.com/opf/openproject/pull/3160/files#diff-8c79271c9d2d36c34932fb68d5560378R203), allowing a very  simplest override.